src/docfx/template/TemplateEngine.cs

```diff
@@ -96,7 +108,7 @@ public static TemplateEngine Create(Docset docset)
 
             if (string.IsNullOrEmpty(docset.Config.Template))
             {
-                return null;
+                return new TemplateEngine("", null);
```

Comment:
Is there a default template path in v2? Something like `{docsetDir}/_theme` @superyyrrzz